nupic/bindings/math.i

```diff
@@ -111,22 +111,22 @@ import_array();
 //--------------------------------------------------------------------------------
 %inline {
 
-  inline nupic::Real64 lgamma(nupic::Real64 x)
```

Comment:
the biggest reason was compiler failing with `inline` there. `-flto` should sort the inlining itself (maybe it's just the combination of manual inline & *.i file, i think in cpp files manual inline is ok). 
(it would be better not to inline only if the function was not used at all / too big, maybe the first could be the case..)